pytential/symbolic/mappers.py

```diff
@@ -61,7 +61,7 @@ def rec_int_g_arguments(mapper, expr):
             name: mapper.rec(arg) for name, arg in expr.kernel_arguments.items()
             }
 
-    changed = (
+    changed = not (
```

Comment:
I do not know how this didn't break the world before.. :facepalm:

Reply:
Agreed. Scary... Thanks for finding it!